README.md

```diff
@@ -1,31 +1,75 @@
-# Dynatrace OneAgent collection
-In its current state, the collection consists of a single role that deploys Dynatrace OneAgent on Linux and Windows operating systems using dedicated configuration and ensures the OneAgent service maintains a running state.
+# Dynatrace OneAgent Ansible collection
+
+Ansible collection for deploying Dynatrace OneAgent.
+
+## Description
+
+Dynatrace OneAgent is responsible for collecting all monitoring data within your monitored environment. A single
```

Comment:
So extensive description seem redundant. Most of it does not describe the collection itself.